src/pybind/mgr/hello/module.py

```diff
@@ -93,34 +76,43 @@ def config_notify(self):
                     self.get_ceph_option(opt))
             self.log.debug(' native option %s = %s', opt, getattr(self, opt))
 
-    def handle_command(self, inbuf, cmd):
+    # there are CLI commands we implement
+    @CLIReadCommand('hello')
+    def hello(self, person_name: Optional[str] = None) -> HandleCommandResult:
+        """
+        Say hello
+        """
+        out = ''
+        if person_name is None:
+            out = 'Hello, ' + cast(str, self.get_module_option('place'))
+        else:
+            out = f'Hello, {person_name}'
+        if self.get_module_option('emphatic'):
+            out += '!'
```

Comment:
Why not promoting adoption of f-strings vs. concat?
```suggestion
        out = f"Hello, {person_name or self.get_module_option('place')}{'!' if emphatic else ''}"
```

Reply:
thanks. add a separate commit to promote f-strings.

Reply:
I somewhat don't like `{... if ... else ''}` constructs within f-strings. I'm ok with them, but they feel a bit clunky

Reply:
neither do i, i ended up with
```py
        fin = '!' if self.get_module_option('emphatic') else ''
        return HandleCommandResult(stdout=f'Hello, {who}{fin}')
```